platform/darwin/src/MGLForegroundStyleLayer.h

```diff
@@ -11,9 +11,8 @@ NS_ASSUME_NONNULL_BEGIN
  `MGLForegroundStyleLayer` is an abstract superclass for style layers whose
  content is defined by an `MGLSource` object.
 
- Do not create instances of this class directly, and do not create your own
- subclasses of this class. Instead, create instances of `MGLRasterStyleLayer`
- and the concrete subclasses of `MGLVectorStyleLayer`.
+ Create instances of `MGLRasterStyleLayer` and the concrete subclasses of
+ `MGLVectorStyleLayer` in order to use `MGLForegroundStyleLayer`'s methods.
```

Comment:
The previous wording (“do not create instances of this class”) matches [Apple documentation](https://developer.apple.com/reference/mapkit/mkdirectionsresponse?language=objc). It’s fine to start with a positive description of what to do, but we still need to emphasize what not to do, because it’s very easy in Objective-C to fall into the trap of initializing an abstract superclass directly.

Additionally, it’s important to state that the developer shouldn’t create their own subclass of this class, due to how this class keeps track of a C++ object under the hood.